packages/create-webstone-app/template/package.json

```diff
@@ -7,10 +7,18 @@
   ],
   "scripts": {
     "dev": "pnpm --recursive --parallel dev",
-    "preinstall": "npx only-allow pnpm"
+    "preinstall": "npx only-allow pnpm",
+    "test": "pnpm test:unit",
+    "test:unit": "c8 --all --include=**/src --reporter=html pnpm test:unit:only",
+    "test:unit:only": "NODE_OPTIONS='--loader tsx' uvu packages tests"
```

Comment:
Having the unit tests inside the `tests` direcotry clashes with the Sveltekit-Playwright Integration, we need to decide where we want unit-tests to be stored I guess

Reply:
I'm not sure if we need tsx here 🤔 . Probably regular `esm` might work fine as well